modules/python/src2/cv2_numpy.hpp

```diff
@@ -18,7 +18,7 @@ class NumpyAllocator : public cv::MatAllocator
     const cv::MatAllocator* stdAllocator;
 };
 
-extern NumpyAllocator g_numpyAllocator;
+inline NumpyAllocator& GetNumpyAllocator() {static NumpyAllocator gNumpyAllocator;return gNumpyAllocator;}
```

Comment:
Why is implementation here and not in .cpp file?
Not sure if we want multiple instances of this object.

---

> GetNumpyAllocator

OpenCV function names start with lowercase.

Reply:
There won't be multiple allocator instances, because function is not marked  as `static`, so linker should strip its all internal `static` variables to 1